inc/reception.class.php

```diff
@@ -549,6 +579,13 @@ static function processMassiveActionsForOneItemtype(MassiveAction $ma, CommonDBT
    }
 
 
+   public function showTransferForm($params = []) {
+      echo "<label class='order_ma'>".__("Desired Entity", "order") . "</label>";
+      Entity::Dropdown(['name' => "entities_id"]);
```

Comment:
Dropdown should be filtered to display only entities corresponding to the order visibility.